mailer.js

```diff
@@ -321,7 +334,8 @@ module.exports = function (log) {
         link: link,
         oneClickLink: oneClickLink,
         supportUrl: this.supportUrl,
-        supportLinkAttributes: this._supportLinkAttributes()
+        supportLinkAttributes: this._supportLinkAttributes(),
+        passwordChangeUrl: this._initiatePasswordChange()
```

Comment:
Wow that's a lot of identical changes...I wonder if we should factor these links out into some common set of attributes that are included in every render, so that we can manage them in a single place.